docs/protocol/src/crypto/decaf377/decoding.md

```diff
@@ -1,24 +1,25 @@
 # Decoding
 
-Decoding to a point works as follows:
+Decoding to a point works as follows where $a$ and $d$ are the curve parameters as described [here](../decaf377.md#curve-parameters).
 
-1. Decode `s_bytes` to a field element $s$, rejecting if the encoding is
-non-canonical.  (If the input is already a field element in the circuit case,
-skip this step).
+1. Decode `s_bytes` to a field element $s$. We interpret these bytes as unsigned
+little-endian bytes. We check if the length has 32 bytes, even though
+the top 3 bits of the last byte are not used. The 253 bits are verified to be canonical, and rejected if not (If the input is already a field element in the circuit case, skip this step).
```

Comment:
We probably shouldn't do this, though, right? The new text is maybe a more accurate description of our current behavior, but shouldn't we enforce that the the entire encoding is canonical, and not leave three bits of malleability at the top?

Reply:
Ah, yeah this describes what we (well Arkworks) currently do. Should we be requiring that the last three bits are zero?

Reply:
Yeah, we should keep the written description, and change the implementation to match it, by checking that the top three bits are zero before passing in to Arkworks.

Reply:
Added this behavior in https://github.com/penumbra-zone/decaf377/pull/34/commits/c1db5ad55290f62584cd6cc1d5e74ae534cff127 and updated the text here